<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmulatorSettingsDialog</class>
 <widget class="QDialog" name="EmulatorSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Model Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Model</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="leMaxCapacity"/>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QComboBox" name="cbModelName"/>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>V</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="leMinVoltage"/>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>mAh</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Min Voltage</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Max Capacity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Max Voltage</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="leMaxVoltage"/>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>V</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Model Data</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="tvBatteryModel"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="btnChangeName">
          <property name="text">
           <string>Change Model Name</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Save to Json</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="btnLoad">
          <property name="text">
           <string>Load from Json</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="btnSaveToDB">
          <property name="text">
           <string>Save to DB</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
